documentation/index.md

```diff
@@ -367,38 +388,45 @@ def pipe():
 debug_has_run = False
 pipe()
 assert debug_has_run == False
+## export RUN_DEBUG_NODES=True
```

Comment:
can you add a comment before this line (something along the lines):
`# you can try running debug nodes `